packages/fast-components-react-base/src/typography/typography.tsx

```diff
@@ -48,8 +48,8 @@ class Typography extends Foundation<ITypographyHandledProps & IManagedClasses<IT
      * Generates class names based on props
      */
     protected generateClassNames(): string {
-        const classes: string = this.props.typeLevel ?
-            get(this.props, `managedClasses.typography__${this.props.typeLevel}`) : get(this.props, "managedClasses.typography__1");
+        const classes: string = this.props.size ?
```

Comment:
nit - can you put the colon and the value following it to a new line to align with other multi-line ternary statements?

Reply:
Sure